docs/feature_tables/mysql.md

```diff
@@ -0,0 +1,75 @@
+# MySQL
+
+Feathub provides `MySQLSource` to read data from a MySQL table and `MySQLSink` to 
+materialize feature view to a MySQL table. Currently, `MySQLSource` can only be used as 
+an online store source.
+
+## Supported Processors and Usages
+
+- Flink: Online Store Source, Append Sink, Upsert Sink
+
+## Example
+
+Here are the examples of using `MySQLSource` and `MySQLSink`:
+
+### Use as Sink
```

Comment:
Would it be useful to mention Append/Upsert to match the content in `Supported Processors and Usages`?

Reply:
The sink can be in append mode or upsert mode in the given example, depending on whether the given feature view has keys. I updated the docs accordingly.